client/components/Help/HelpPortalModal/HelpPortalModal.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from 'react'
+import HelpPortalModal from './HelpPortalModal'
+
+export default { title: 'Help/HelpPortalModal/HelpPortalModal' }
+
+export const Default = () => {
+  const [el, setEl] = useState<HTMLElement | null>(null)
+
+  return (
+    <div>
```

Comment:
This element should be replaced with a React fragment(`<>`).

Reply:
done.